storage/innobase/xtrabackup/src/xbcloud/s3.h

```diff
@@ -87,6 +92,13 @@ class S3_signerV4 : public S3_signer {
 
   void sign_request(const std::string &hostname, const std::string &bucket,
                     Http_request &req, time_t t) override;
+  void update_keys(const std::string &_access_key,
+                   const std::string &_secret_key,
+                   const std::string &_session_token) override {
+    access_key = _access_key;
+    secret_key = _secret_key;
+    session_token = _session_token;
```

Comment:
[optional]
we would use m_ for members and the parameters would be passed as is without _

Reply:
This will be a big refactor on S3 client. V3 and V4 clients have member declared as `access_key` without the m_ prefix and to be consistent, we should refactor Swift and Azure clients too.